src/SpeedDial/SpeedDial.tsx

```diff
@@ -0,0 +1,41 @@
+import { ReactNode, useState } from "react";
+import Add from "../icons/Add";
+import { ButtonGroup } from "../ButtonGroup";
+import { st, classes } from "./speeddial.st.css";
+import { TooltipButton } from "../TooltipButton";
+
+interface SpeedDialProps {
+  children: ReactNode;
+  tooltipTitle: string;
+}
+
+function SpeedDial(props: SpeedDialProps) {
+  const [speedDialOpen, triggerSpeedDial] = useState("false");
+  const { children, tooltipTitle } = props;
+
+  const addButton = (
+    <TooltipButton
+      buttonProps={{
+        onBlur: () => triggerSpeedDial("false"),
+        onClick: () =>
+          triggerSpeedDial(speedDialOpen === "false" ? "true" : "false"),
+        icon: <Add />,
+        className: classes.addButton,
+      }}
+      tooltip={tooltipTitle}
+    />
+  );
+
+  const buttonGroup = (
+    <ButtonGroup className={classes.buttonGroup}>{children}</ButtonGroup>
```

Comment:
Why not just use the <ButtonGroup> inside the <div> instead of creating a variable since only used once?